Lib/calendar.py

```diff
@@ -832,7 +841,12 @@ def formatstring(cols, colwidth=_colwidth, spacing=_spacing):
 
 
 def timegm(tuple):
-    """Unrelated but handy function to calculate Unix timestamp from GMT."""
+    """
```

Comment:
I think it's fine to have a short description. The full description is in https://docs.python.org/3/library/calendar.html#calendar.timegm.